content/en/docs/project-user-guide/application-workloads/routes.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Routes"
+keywords: "KubeSphere, Kubernetes, Route, Ingress"
+description: "How to create a Route in KubeSphere"
+weight: 10270
+---
+
+This document describes how to create, use, and edit a Route on KubeSphere.
+
+A Route on KubeSphere is the same as an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/#what-is-ingress) on Kubernetes. You can use a Route and a single IP address to aggregate and expose multiple services.
+
+## Prerequisites
+
+- You need to create a workspace, a project and an account (for example, `project-admin`). The role of the account in the project must be `admin`. For more information, see [Create Workspaces, Projects, Accounts and Roles](/docs/quick-start/create-workspace-and-project/).
```

Comment:
You need to create a workspace, a project and an account (for example, `project-admin`). The role of the account in the project must be `admin`.
->
You need to create a workspace, a project and two accounts (for example, `project-admin` and `project-regular`). `project-admin` must be granted the role of `admin` in the project and invite `project-regular` to the same project with the role of `operator`.

Please note that for the operation in reality, `project-regular` should be the one to create a Route.